app/auto-eligibility/processing/update-access-granted.js

```diff
@@ -0,0 +1,18 @@
+const db = require('../../data')
+const sequelize = require('sequelize')
+const upperLimit = require('../../auto-eligibility/config').waitingListScheduler.upperLimit
+const waitingListQuery = `(SELECT sbi FROM eligibility WHERE waiting_updated_at IS NOT NULL AND access_granted = false ORDER BY waiting_updated_at ASC LIMIT ${upperLimit})`

```

Comment:
discussion comment. Should not be susceptible to SQL injection as the value injected into the string is from config and not user input in any way.

Reply:
Agreed and to be picked up as part of https://github.com/DEFRA/ffc-ahwr-eligibility/pull/15

FYI @marcinmo-kainos